src/main/java/org/lobid/resources/run/WikidataGeodata2Es.java

```diff
@@ -70,7 +70,7 @@
 
 				{
 					put("99", "Q22865 Q262166 Q253019 Q1852178 Q15632166");
-					put("97", "Q106658");
+					put("97", "Q106658 Q5283531");
 					put("96", "Q829277");
 					put("36", "Q3146899 Q2072238");
 					put("97", "Q106658");
```

Comment:
Oh, there are _two_ entries of `97`, which is not good. As long as the entries are same this doesn't have an effect. Now, as they are different, it does behave differently - it overwrites the first entry with the second entry, ignoring "Q5283531".